src/components/00-materials/styles/variables.scss

```diff
@@ -19,3 +19,4 @@ $spacing-9: 120px;
 
 // shadow
 $box-shadow-1: 0px 1px 10px rgba(0, 0, 0, 0.15);
+$box-shadow-2: 0px 1px 20px rgba(0, 0, 0, 0.15);
```

Comment:
changed it to `$box-shadow-default` & `high` since these are the names in the sigma [https://www.figma.com/file/6zurYk3bJpzUg0H2THSxGF/AXA-UI-Kit?node-id=6138%3A1](url)